tests/settings.py

```diff
@@ -35,6 +35,7 @@
         'rele.contrib.LoggingMiddleware',
         'rele.contrib.DjangoDBMiddleware',
     ],
+    'FILTER_SUBS_BY': lambda attrs: attrs.get('type') == 'foo'
```

Comment:
Is this properly working? I see in the examples the callback is declared with `attrs` being a keyword parameter: 
```python
    def subscriptions(self):
        return load_subscriptions_from_paths(
            ['tests.test_config'],
            sub_prefix='test',
            filter_by=lambda **attrs: attrs.get('lang') == 'en')
```

Is this something that can make this global filter fail? 🤔

Reply:
We are not using the settings.py so I'm gonna remove the filter from there. Also I'll change the filter_by in the subscription fixture.